src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,12 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllCinemaHallsQuery = session.createQuery(
+                    " FROM Movie ", Movie.class);
+            return getAllCinemaHallsQuery.getResultList();
```

Comment:
The variable `getAllCinemaHallsQuery` is misleading in the context of the `MovieDaoImpl` class. Consider renaming it to something more appropriate, like `getAllMoviesQuery`, to reflect that it is querying movies, not cinema halls.